apps/server/src/controllers/integrationController.ts

```diff
@@ -272,6 +272,7 @@ export function dispatchFromAdapter(
     // WS: {type: 'change', payload: { eventId, property, value } }
     case 'change': {
       const { eventId, property, value } = payload as ChangeOptions;
+      //TODO: cue can't pass this check
```

Comment:
I could use a tip on who to get the cue filed past this check

Reply:
makes sense. it seems that we are using a placeholder event to check this. since we are unable to provide a placeholder cue this fails.

We could
a) add a cue field to the placeholder and make sure it gets overriden on creation of the event

b) modify this check to pass if the field is cue like
` if (property === "cue" ||  !isKeyOfType(property, event)) `

a might be more maintainable, but also more work

Reply:
The issue you find here is that cue is not part of the object that you use to check against

We do not want to modify that object since it breaks our rules, could we do the second check?
if (property === "cue" ||  !isKeyOfType(property, event))